android/gradle.properties

```diff
@@ -39,5 +39,6 @@ POM_DEVELOPER_ID=revenuecat
 POM_DEVELOPER_NAME=RevenueCat, Inc.
 #Do not sign releases. When calling uploadArchives pass -PRELEASE_SIGNING_ENABLED=true
 RELEASE_SIGNING_ENABLED=false
+ANDROID_VARIANT_TO_PUBLISH=bc8Release
```

Comment:
This is just a name, right? I mean, it's not referencing anything else declare in some other place

Reply:
Well, it's declaring the final gradle variant that needs to be published by default. In this case, it will be the variant `bc8Release`, which is basically flavor `bc8` + build type `release`. The publishing plugin will use this variant by default when publishing, if no other parameters are passed.